perfkitbenchmarker/linux_benchmarks/cloudharmony_network_benchmark.py

```diff
@@ -87,6 +94,8 @@
                      'threads.')
 flags.DEFINE_integer('ch_network_tcp_samples', 10, 'The number of test samples '
                      'for TCP tests (rtt, ssl or ttfb).')
+flags.DEFINE_enum('ch_ssl_encryption_type', ECC, [ECC, RSA, UNENCRYPTED],
```

Comment:
Can we try out the new DEFINE_enum_class
https://github.com/abseil/abseil-py/blob/main/absl/flags/_defines.py#L459

Reply:
Changed to enum class